Exercise7/src/IO/CharCopy.java

```diff
@@ -0,0 +1,52 @@
+package IO;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+public class CharCopy {
+	public static void main(String[] args){
+		File source = new File("source.txt");
+		try {
+			source.createNewFile();
+		} catch (IOException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		}
+		File destination = new File("outputFile.txt");
+		FileInputStream inputStream = null;
+		try {
+			inputStream = new FileInputStream(source);
+		} catch (FileNotFoundException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		}
+		FileOutputStream outputStream = null;
+		try {
```

Comment:
It's fine to have an overall try-catch block instead of creating one for each statement that might cause an error. And, add a `finally` block to close the streams if they are open